tests/testing/test_model_builder.py

```diff
@@ -14,6 +16,12 @@
 )
 
 
+class TableWithArrayField(Table):
+    strings = Array(Varchar(30))
+    integers = Array(Integer())
+    floats = Array(Real())
```

Comment:
i have constructed this here as i didn't necessarily want to edit the example tables and invoke a bunch of documentation changes et cetera! let me know if you'd prefer an array field on an existing example table.

Reply:
That's fine 👍